wls-wahlvorstand-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorstandservice/security/AuthenticationHandler.java

```diff
@@ -0,0 +1,11 @@
+package de.muenchen.oss.wahllokalsystem.wahlvorstandservice.security;
+
+import java.util.Optional;
+import org.springframework.security.core.Authentication;
+
+public interface AuthenticationHandler {
```

Comment:
Nachdem wir das Interface ein zweiten Mal haben könnten wir es in wls-common nachrüsten. Bitte ein Issue dazu erstellen.